assets/together.yaml

```diff
@@ -109,3 +109,70 @@
   prohibited_uses: ''
   monitoring: ''
   feedback: ''
+
+- type: dataset
+  name: RedPajama-Data-1T
+  organization: Together
```

Comment:
I think this should be a range of organizations based on the collaboration described in the blog post